projects/plugins/jetpack/extensions/blocks/ai-assistant/lib/prompt/index.ts

```diff
@@ -104,6 +104,87 @@ ${ postTitle?.length ? `- Current title: ${ postTitle }\n` : '' }${
 	};
 }
 
+type PromptOptionsProps = {
+	content: string;
+	language?: string;
+	tone?: ToneProp;
+	role?: PromptItemProps[ 'role' ];
+};
+
+function getCorrectSpellingPrompt( {
+	content,
+	role = 'user',
+}: PromptOptionsProps ): Array< PromptItemProps > {
+	return [
+		{
+			role,
+			content: `Repeat the following text, correcting any spelling and grammar mistakes:\n\n${ content }`,
+		},
+	];
+}
+
+function getSimplifyPrompt( {
+	content,
+	role = 'user',
+}: PromptOptionsProps ): Array< PromptItemProps > {
+	return [
+		{
+			role,
+			content: `Simplify the following text, using words and phrases that are easier to understand. Write the content in the same language as the original content:\n\n${ content }`,
```

Comment:
Why not a rule to avoid repeating it?

Reply:
because sometimes we don't want to preserve the language and the tone. For instance, when changing the language and/or the tone :-D